src/flow/Promise.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
If we do this here, please move this file to `<root>/flow/`, this isn't compatible with the build process.